multiversx/mx-chain-go
# Upgrade indexer ( frozen esdts tokens)

## Reasoning behind the pull request
- Changed the encoding for `tokenProperties` from `outport` package
- Updated indexer version
## Proposed changes
- Upgrade the version of the `mx-chain-es-indexer` module : 
  PR from indexer module: https://github.com/multiversx/mx-chain-es-indexer-go/pull/200

## Testing procedure
a. Create an ESDT token ( 'fungible' type). 
b. Send some ESDT tokens to a new address 
c. Freeze ESDT tokens of that address 
d. Check if in `accountsesdt` index field `frozen` is present 

Do the same steps for an NFT. 

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
